augly/audio/functional.py

```diff
@@ -454,39 +454,28 @@ def high_pass_filter(
         func_kwargs = deepcopy(locals())
         func_kwargs.pop("metadata")
 
-    rc = 1 / (2 * math.pi * cutoff_hz)
-    dt = 1 / sample_rate
-    alpha = rc / (rc + dt)
     num_channels = 1 if audio.ndim == 1 else audio.shape[0]
+    audio = audio.reshape((num_channels, -1))
 
-    if num_channels == 1:
-        audio = audio.reshape(1, audio.shape[0])
-
-    frame_count = audio.shape[1]
-    high_pass_array = np.zeros(audio.shape)
-
-    for i in range(num_channels):
-        high_pass_array[i][0] = audio[i][0]
-        for j in range(1, frame_count):
-            high_pass_array[i][j] = alpha * (
-                high_pass_array[i][j - 1] + audio[i][j] - audio[i][j - 1]
-            )
+    aug_audio, out_sample_rate = sox_effects.apply_effects_tensor(
+        torch.Tensor(audio), sample_rate, [["highpass", str(cutoff_hz)]]
+    )
 
-    if num_channels == 1:
-        high_pass_array = high_pass_array.reshape((high_pass_array.shape[1],))
+    high_pass_array = aug_audio.numpy()
+    if audio.shape[0] == 1:
+        aug_audio = aug_audio.reshape((aug_audio.shape[-1],))
```

Comment:
```suggestion
        high_pass_array = high_pass_array.reshape((aug_audio.shape[-1],))
```
+1 to bug found in https://github.com/facebookresearch/AugLy/pull/153#pullrequestreview-823013558

Reply:
I should also be calling `high_pass_array` in the .shape(), right?